src/Common/tests/Tests/System/Net/HttpKnownHeaderNamesTest.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Justin Van Patten. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
```

Comment:
nit: Add blank line to separate top level 'using' groups (as per CoreFx coding guidlines).